src/components/govukComponents/Select.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import Label from './Label'
+import Hint from './Hint'
+import ErrorMessage from './ErrorMessage'
+
+function Select(props) {
+
+  var describedBy = props.describedBy
+  if (props.hint) {
+    const hintId = props.id + '-hint'
+    describedBy += ' ' + hintId
+    var hint = <Hint
+      id={hintId}
+      {...props.hint}
+    />
+  }
+
+  if (props.errorMessage) {
+    const errorId = props.id ? props.id + '-error' : '';
+    describedBy += ' ' + errorId
+    var errorMessage = <ErrorMessage
+      id={errorId}
+      {...props.errorMessage}
+    />
+  }
+
+  if (props.items.selected) {
```

Comment:
@mick-jones `props.items.selected` isn't a _thing_ is it? `props.items` is an array of objects, of which `selected` is a property of one of the objects. Shouldn't this probably be doing something like this:

```
const selectedItem = props.items.find(item => item.selected === true)
const defaultValue = selectedItem.value